integration-tests/tests/run-analysis.js

```diff
@@ -29,8 +29,11 @@ const testRunAnalysisFn = _.flow(
     action: () => findText(page, 'A cloud environment consists of application configuration, cloud compute and persistent disk(s).')
   })
 
+
   await click(page, clickable({ textContains: 'Close' }))
 
+  await delay(3000)
```

Comment:
It would be to add comments explaining why all these delays are necessary. What is not handled by the next step's timeout?